src/languages/en-EN.json

```diff
@@ -50,8 +50,8 @@
   "MUSIC_PAUSED": "Music is already paused, did you mean `resume`?",
   "MUSIC_RESUME": "Music has been resumed.",
   "MUSIC_PLAYING": "Music is already playing, did you mean `pause`?",
-  "VOICE_PERSIST": "There are no more tracks to play, add more using request.",
-  "QUEUE_END": "There are no more tracks in the queue to play, leaving voice.",
+  "VOICE_PERSIST": "Your queue is empty, add more using request.",
```

Comment:
This could use some additional context. Perhaps stating "using the `request` command" would be prudent.